tests/integration/test_dataset.py

```diff
@@ -6,20 +6,15 @@
 from e3sm_diags.derivations import acme as acme_derivations
 from e3sm_diags.driver.utils.dataset import Dataset
 from e3sm_diags.parameter.core_parameter import CoreParameter
-
-
-def get_abs_file_path(relative_path):
-    return os.path.join(os.path.dirname(os.path.abspath(__file__)), relative_path)
```

Comment:
This function is no longer necessary since we replace it with an absolute path (`TEST_DATA_PATH`).